R/get-net-task.R

```diff
@@ -0,0 +1,18 @@
+#' Returns an instace of the specified .NET Task
+#'
+#' @param taskName The name of the task to retrieve (without the Get)
+#'
+#' @return An instance of the Task
+#'
+#' @details
+#' simulationLoader <- getNetTask("SimulationLoader")
+getNetTask <-function(taskName) {
+  rClr::clrCallStatic("OSPSuite.R.Api", paste0("Get", taskName))
+}
+
+getContainerTask <- function() {

```

Comment:
This is where I am doing the cash. As a caller, we have to know if we need to cache stuff or not
I would argue this should not be done too often. For example, SimulationLoader. We need one new instance for each to ensure that all dependencies are cleanup properly. Too much optimziation can lead to brutal bug.

I am still not sure to be honest that we need to save a containerTask reference but we have the concept now so we can play with it